### PR TITLE
Mark

### PR DESCRIPTION
**Pull request by marktvbui**

### CODE REVIEW

**Comment by marktvbui** [comment]
dried up my code, please review and approve merge

File: assets/javascript/petapp.js [hunk 1]
```diff
@@ -276,7 +271,7 @@ $(document).ready(function() {
   // this retrieves business data
   function getBizData() {
     var search = $("#selection-input").val();
-    var location = $("#address-input").val();
+    var location = $("#pac-input").val();
```
Comment (kristalane) [inline assets/javascript/petapp.js:274]: Hey, Mark - is there a reason you wanted to revert this to #pac-input? "address-input" seemed more easily understandable to me... but maybe just a matter of when you last pulled :)

File: assets/javascript/petapp.js [hunk 2]
```diff
@@ -301,14 +296,16 @@ $(document).ready(function() {
         var phone = response.listings[i].phone.dispNum;
         var geoCode = response.listings[i].geoCode.latitude
           + response.listings[i].geoCode.longitude;
+        $('#displayGif').prepend(selectionDiv);
         var result = $("<p>")
           .html(name + "<br>" + "Address: " + address + "<br>" + "Phone: " + phone + "<br>" + "Website: " + resultUrl)
-          .appendTo($("#displayAPI"));
+          .appendTo($("#displayGif"));
```
Comment (kristalane) [inline assets/javascript/petapp.js:302]: Hmm... where did your giphy API function go? It seems like you've merged the Yellow API and giphy API functions, but maybe github is only showing me part of the codebase in this review.

File: assets/javascript/petapp.js (hunk 1 above)
Comment (kristalane) [reply assets/javascript/petapp.js:274]: Resolving - sounds like "pac-input" is a specification for Google Maps API functionality, so we'll keep it for now.

**Review by kristalane — APPROVED** [review]

**Pull request merged**